e2e-spock-geb/files/build.gradle

```diff
@@ -0,0 +1,181 @@
+    testCompile "com.konghq:unirest-java:${unirestVersion}"
+    testCompile "org.gebish:geb-spock:${gebVersion}"
+    testCompile "org.seleniumhq.selenium:selenium-firefox-driver:${seleniumVersion}"
+    testCompile "org.seleniumhq.selenium:htmlunit-driver:${htmlunitVersion}"
+    testRuntime "org.seleniumhq.selenium:selenium-support:${seleniumVersion}"
+}
+
+sourceSets {
+    installation {
+        groovy {
+            srcDir 'src/test/installation/groovy'
+        }
+        java {
+            srcDir 'src/test/installation/java'
+        }
+        resources {
+            srcDir 'src/test/resources'
+        }
+        compileClasspath += sourceSets.main.output + sourceSets.test.output + configurations.testRuntime
+        runtimeClasspath += sourceSets.main.output + sourceSets.test.output + configurations.testRuntime
+    }
+
+    integration {
+        groovy {
+            srcDir 'src/test/integration/groovy'
+        }
+        java {
+            srcDir 'src/test/integration/java'
+        }
+        resources {
+            srcDir 'src/test/resources'
+        }
+        compileClasspath += sourceSets.main.output + sourceSets.test.output + configurations.testRuntime
+        runtimeClasspath += sourceSets.main.output + sourceSets.test.output + configurations.testRuntime
+    }
+
+    acceptance {
+        groovy {
+            srcDir 'src/test/acceptance/groovy'
+        }
+        java {
+            srcDir 'src/test/acceptance/java'
+        }
+        resources {
+            srcDir 'src/test/resources'
+        }
+        compileClasspath += sourceSets.main.output + sourceSets.test.output + configurations.testRuntime
+        runtimeClasspath += sourceSets.main.output + sourceSets.test.output + configurations.testRuntime
+    }
+}
+
+configurations {
+    installationCompile.extendsFrom testCompile
+    installationRuntime.extendsFrom testRuntime
+
+    integrationCompile.extendsFrom testCompile
+    integrationRuntime.extendsFrom testRuntime
+
+    acceptanceCompile.extendsFrom testCompile
+    acceptanceRuntime.extendsFrom testRuntime
+}
+
+class TestExecutionPhases {
+    static final String INSTALLATION = "installation"
+    static final String INTEGRATION = "integration"
+    static final String ACCEPTANCE = "acceptance"
+}
+
+class TestLanguages {
+    static final String JAVA = "java"
+    static final String GROOVY = "groovy"
+}
+
+def generateTaskName(def type, def language) {
+    return "${type}-${language}"
+}
+
+// Task to create in a parametrized way Tests task
+def executeTest(def type, def language) {
+    return tasks.create(generateTaskName(type, language), Test) {
+        description = "Runs ${type} tests ${language}."
+        group = "verification"
+
+        if ("${language}".equalsIgnoreCase(TestLanguages.JAVA)) {
+            // Mandatory to run JUnit 5 test but incompatible with groovy tests
+            useJUnitPlatform()
+        }
+
+        ignoreFailures = Boolean.getBoolean("test.ignoreFailures")
```

Comment:
@hrcornejo let's set this to `true` by default and add a comment on top what the user can control here.

Reply:
Done